scripts/jenkins/Jenkinsfile

```diff
@@ -838,6 +841,19 @@ def buildCheckReleaseNotes() {
   }]
 }
 
+/* Stage checking if for reuse compliancy */
+def buildCheckReuseCompliant() {
+  def stageName = "check-reuse-compliant"
+  return [(stageName): {
+    maybeStage(stageName, !isMaster()) {
+      withDockerEnv(DOCKER_IMAGES.bullseye, [DockerOpts.MOUNT_MIRROR]) {
```

Comment:
make sure that you also install reuse in the bullseye container: scripts/docker/debian/bullseye/Dockerfile

Reply:
nice catch. Installation has been added.